### DISCUSSION TITLE
Welcome to Funkin Discussions! (IMPORTANT, READ!!!)

### DISCUSSION THREAD

**Discussion by ninjamuffin99** [FunkinCrew/Funkin#1478] [Announcements]
## 👋 GITHUB DISCUSSIONS REAL!
Getting around to tidying this dang github place up I think. THESE ARE GITHUB DISCUSSIONS!!! Basically like some forum bullshit, hopefully to keep the issues tab a bit less cluttered with questions that can be answered, as well have a nice filtering system with upvotes like some reddit bullshit.

Discussions should be around FRIDAY NIGHT FUNKIN' **CODE**!!!! This isn't really a place to ask ninjamuffin99 or devs some wacky quirky obnoxious meme question, or to ask about kickstarter rewards. Moreso about asking questions that EACH OTHER can answer if that makes sense.

Maybe you need help compiling, maybe you have an idea, maybe you have a question about what code does what, and why it's there. Feel free to ask them here. This is all fresh and I'm willing to modify things as needed to fit FUNKIN' needs, and hopefully won't just be a giant waste of time.

  FEW THINGS
  * Don't be overly obnoxious
  * Be cool
  * Engage with other community members.

This is early on, I don't want this to be something I have to be constantly moderating, I would genuinely rather be working on code, but I think if done right, this can be worthwhile. Try to maintain an essence of being cool and kind to newcomers. Not everyone is a master of code or programming. It could be some kids very first time learning about Haxe and compiling and everything. Be as accepting as possible to all kinds of genuine questions that you can. Everything, by Everyone.
 
![misato](https://user-images.githubusercontent.com/22229331/130865349-96af7c77-1bc5-4697-a8c5-aff5571b1946.png)

-ninjamuffin99

**Comment by ninjamuffin99** [comment]
inb4 luis

**Reply by ghost** [reply]
WAIT IS THAT YOU

**Reply by RevixDev** [reply]
Furry gang?

**Comment by htv04** [comment]
Yo this is awesome, thanks for setting this up

**Reply by ninjamuffin99** [reply]
tnx x3

**Comment by santumerino** [comment]
aight sick

**Comment by Cheemsandfriends** [comment]
I've got a question.
Have you considered funkin Discussions in the final Product?
For feedback, bugs thing and stuff

**Reply by ninjamuffin99** [reply]
yeah, these are DEFINITELY gonna be useful post-launch of updates and the general release

**Reply by suprstarrd** [reply]
I read this as "Funkin Discord" and instantly shrieked in fear

**Reply by isakube** [reply]
> I read this as "Funkin Discord" and instantly shrieked in fear

too much tos breaking :pensive:

**Reply by RevixDev** [reply]
To be honest I recon it would be cool if we could make discussions via the game itself. But does it look like we know how to network? No.

**Comment by Infamous150** [comment]
week 7 on desktop in sept.?

**Reply by Cheemsandfriends** [reply]
no

**Reply by Beelijah** [reply]
it took 3 more years

**Comment by AbnormalPoof** [comment]
fnf mod

**Comment by Dark-Tails53** [comment]
When will Week 8 be released?

**Reply by AbnormalPoof** [reply]
never

**Reply by ghost** [reply]
hank is week 8 /cb

**Reply by Vovo4k** [reply]
Idk 🥶

**Reply by Beelijah** [reply]
3 more years

**Reply by Beelijah** [reply]
6 more years

**Reply by charlesisfeline** [reply]
9 more years

**Reply by Beelijah** [reply]
42 more years

**Reply by charlesisfeline** [reply]
100 more years

**Reply by Lasercar** [reply]
Next update

fnever

**Reply by josh2750** [reply]
when GTA 6 is released

**Reply by realvirtu** [reply]
tomorrow

**Comment by AbnormalPoof** [comment]
Hey muffin, have you considered updating Playstate.hx to use `FlxG.cameras.setDefaultDrawTarget` instead of `FlxCamera.defaultCameras`?

**Comment by isakube** [comment]
dude

Mercury's releasing in 2 weeks
Geometry Dash 2.2 is releasing in 3-6 months,
and Discussions are enabled on ninjamuffin99/Funkin

**Reply by ghost** [reply]
> Geometry Dash 2.2 is releasing in 3-6 months,

Do you mean 36 years?

**Reply by Beelijah** [reply]
526*

**Comment by Infamous150** [comment]
how's it goin mr. muffin?

**Comment by GStudiosX2** [comment]


**Reply by ACrazyTown** [reply]
When the next demo releases, however there's a decompilation of Week 7 available that might be useful: https://github.com/AngelDTF/FNF-NewgroundsPort

**Reply by ACrazyTown** [reply]
> when is the next demo though? but okay

There's no release date, when it comes it comes. The team's trying to polish it as much as they can

**Comment by fnfmod-maker** [comment]
I need help with making the notes. Is anyone able to help me?
